src/table/utils/handle-key-press.ts

```diff
@@ -231,7 +245,10 @@ export const handleBodyKeyDown = ({
       focusSelectionToolbar(evt.target as HTMLElement, keyboard, evt.shiftKey);
       break;
     case 'c':
-      isFlagEnabled('PS_15585_SN_TABLE_BASIC_FEATURES') && (evt.ctrlKey || evt.metaKey) && copyCellValue(cell);
+      {
+        const cellValue = cell.qText;
+        areBasicFeaturesEnabled && (evt.ctrlKey || evt.metaKey) && copyCellValue(cellValue);
```

Comment:
Sorry I meant this
```suggestion
        areBasicFeaturesEnabled && (evt.ctrlKey || evt.metaKey) && copyCellValue(cell.qText;);
```
cellValue is only used once, no need to create a seperate variable.